src/guiapi/src/window_spin.cpp

```diff
@@ -67,3 +68,63 @@ const window_class_spin_t window_class_spin = {
         },
     }
 };
+
+void window_spin_t::SetItemIndex(int idx) {
+    if (count > idx) {
+        index = idx;
+        value = min + step * index;
+    }
+    Invalidate();
```

Comment:
`Invalidate()` in case of `count <= id` as well?

Reply:
I am not sure. It is replaced old code. I did not want to break any functionality.

Reply:
OK. So add FIXME to check it later.